cerbero/tools/osxrelocator.py

```diff
@@ -76,6 +76,14 @@ def change_libs_path(self, object_file):
                                                object_file)
                 shell.call(cmd, fail=False)
 
+    def change_lib_path(self, object_file, old_path, new_path):
+        for lib in self.list_shared_libraries(object_file):
+            if old_path in lib:
+                new_path = lib.replace(old_path, new_path)
+                cmd = '%s -change "%s" "%s" "%s"' % (INT_CMD, lib, new_path,
+                                               object_file)
+                shell.call(cmd, fail=False)
```

Comment:
Shouldn't this `fail=True`? If the path is not changed, it won't work properly after unfreezing